app/components/MovieTrailerPlayer.js

```diff
@@ -5,25 +5,54 @@ import Axios from 'axios';
 class MovieTrailerPlayer extends Component {
   state = {
     currentMovieTrailerUrl: '',
+    movieTitle: '',
+    movieYear: '',
   };
 
   componentDidMount() {
     const { apiUrl, match } = this.props;
     const fullTrailerUrl = `${apiUrl}${match.params.movieId}`;
 
     Axios.get(fullTrailerUrl).then(({ data = {} }) => {
-      this.setState({ currentMovieTrailerUrl: data.trailer });
+      this.setState({
+        currentMovieTrailerUrl: data.trailer,
+        movieTitle: data.title,
+        movieYear: data.year,
+      });
     });
   }
 
   render() {
-    const { currentMovieTrailerUrl } = this.state;
+    const { currentMovieTrailerUrl, movieTitle, movieYear } = this.state;
+    const ytVideoID = currentMovieTrailerUrl.split(
+      /(https?:)(\/\/www\.youtube\.com\/watch\?v=)(\S\D.*)/,
```

Comment:
As an option, if you get rid of the regex this could be easier to read.
``` JS
const ytVideoID = currentMovieTrailerUrl.split("v=")[1].substring(0, 11);
```
Youtube video ID is always the value of `v` and has the same lenght.

Reply:
Youtube videos ID always have the same length ?